tools/BuildAssets/targets/common.targets

```diff
@@ -59,6 +60,27 @@
       <ProjectRootDir/>
   </PropertyGroup>
 
+
+  <Target Name = "ImportAutoRestCodeGenerationModule" AfterTargets="Init">
```

Comment:
@dsgouda any particular reason why why the copying of modules to the user module directory cannot be done in the task, rather than using AfterTargets?
Any advantage of doing the copy as part of a target?
The task is meant to copy files to directory, why not copy modules as part of that?

Reply:
The copy task looks very specific with regards to source and destination directories. We may have to modify the task to look for a particular destination directory just for the powershell modules which defeats the purpose of a generic copy task

Reply:
Move copying to dll and keep the import as a single target